processors/minsev/example_test.go

```diff
@@ -10,39 +10,52 @@ import (
 	"strings"
 	"sync"
 
-	"go.opentelemetry.io/otel/log"
+	"go.opentelemetry.io/contrib/processors/minsev"
+	logapi "go.opentelemetry.io/otel/log"
+	"go.opentelemetry.io/otel/sdk/log"
```

Comment:
Nit, but the (unwritten) convention in other packages is to use the default name for the API, and to name the SDK `logsdk`

Reply:
https://github.com/open-telemetry/opentelemetry-go-contrib/pull/6833/commits/1c28d9d2667fb66433759fec92073ab4633868a4